index.html

```diff
@@ -0,0 +1,169 @@
+
+        <!--Navigation bar-->
+
+        <nav class="fixed">
+
+            <!-- Nav Container -->
+            <ul id="navlist" class="nav-list">
+
+                <li class="logo"><a href="#home">Darion Richards | Portfolio</a></li>
+                <li id="toggle"><span class="bars"></span></li>
+                <ul id="items" class="items">
+                    <li id="navItem1"><a href="#about-me">About Me</a></li>
+                    <li id="navItem2"><a href="#projects">Projects</a></li>
+                    <li id="navItem3"><a href="#contact-me">Contact Me</a></li>
+                </ul>
+            </ul>
+        </nav>
+
+    </header>
+
+    <!--Banner Section-->
+
+    <section class="hero">
+        <!-- <img src="./assets/images/banner-portfolio.jpg" alt="banner-portfolio" /> -->
+    </section>
+
+    <!--Main Content Section-->
+
+    <main>
+
+        <!-- About Me -->
+
+        <section id="about-me" class="container">
+            <section>
+
+                <!-- Heading -->
+
+                <section class="heading">
+                    <h1>About Me</h1>
+                </section>
+
+                <!-- Title -->
+
+                <section class="about-title">
+                    <h2>Title</h2>
+                </section>
+
+                <!-- Info -->
+
+                <section class="about-text">
+                    <p> Lorem ipsum dolor sit amet consectetur adipisicing elit. Nihil natus recusandae amet nostrum odit, ut unde non sint, quia sed inventore odio, delectus iure! Eligendi vitae fuga blanditiis accusamus tempore! Lorem ipsum dolor sit amet
+                        consectetur adipisicing elit. Nihil natus recusandae amet nostrum odit, ut unde non sint, quia sed inventore odio, delectus iure! Eligendi vitae fuga blanditiis accusamus tempore! Lorem ipsum dolor sit amet consectetur adipisicing
+                        elit. Nihil natus recusandae amet nostrum odit, ut unde non sint, quia sed inventore odio, delectus iure! Eligendi vitae fuga blanditiis accusamus tempore!
+                    </p>
+                </section>
+            </section>
+        </section>
+
+        <!-- Projects -->
+
+        <section id="projects" class="project-container">
+            <section>
+
+                <!-- Heading -->
+
+                <section class="heading">
+                    <h1>Projects</h1>
+                </section>
+
+
+                <!-- Projects Container -->
+
+                <section class="project-card-container">
+
+                    <!-- Horiseon Project -->
+
+                    <section class="main-project">
+                        <section class="project-card">
+                            <div class="display-topleft">Horiseon Project (Hover)</div>
+                            <a href="https://github.com/DarionRichards/horiseon-semantic-refactoring"><img class="img-opacity" src="./assets/images/01-html-css-git-homework-demo.png" alt="Horiseon Project Demo Example" /> </a>
```

Comment:
To address the issue of the main project image not resizing for smaller view ports, you can add a class to the `<img>` element so that you can target it directly and adjust the size. Try changing the `width` of this on the smaller viewports.